unstructured/documents/elements.py

```diff
@@ -502,6 +513,7 @@ def field_consolidation_strategies(cls) -> dict[str, ConsolidationStrategy]:
             "link_urls": cls.LIST_CONCATENATE,
             "link_start_indexes": cls.DROP,
             "links": cls.DROP,  # -- deprecated field --
+            "message_id": cls.FIRST,
```

Comment:
@scanny, I wanted to double-check that this was the correct consolidation strategy to use.

Reply:
We would expect the value to be the same for all elements in the element stream, right? This is basically what the `FIRST` strategy is for. It accommodates a situation in which some elements might have `None` for this value, but once it encounters a non-`None` value, the first one is as good as any other (because they're all the same).

Reply:
Yah, they would all be the same